generators/cleanup.js

```diff
@@ -75,6 +77,29 @@ function cleanupOldFiles(generator) {
         generator.removeFile(`${ANGULAR_DIR}admin/metrics/metrics-modal.component.ts`);
         generator.removeFile(`${CLIENT_TEST_SRC_DIR}spec/app/admin/metrics/metrics-modal.component.spec.ts`);
     }
+    if (generator.isJhipsterVersionLessThan('6.3.0')) {
+        generator.removeFile(`${ANGULAR_DIR}account/index.ts`);
```

Comment:
shouldn't we add a check on client framework to restrict this cleanup to angular app?

Reply:
Yes, this is good idea! When doing PR I manually checked that React part have not such files ATM so that should work correctly. But will add this condition to be absolutely sure that no wrong files are deleted.

Reply:
Change done.